### ISSUE TITLE
SortOrder Enum for BinaryNinja causes error

### ISSUE THREAD

**Issue by rvrecen** [gaasedelen/lighthouse#130]
Traceback (most recent call last):
  File "...\plugins\lighthouse\ui\coverage_table.py", line 1058, in sort
    sorted_functions = sorted(
TypeError: 'SortOrder' object cannot be interpreted as an integer

Issue relates to Enum or BinaryNinja api, but solution is to pull value from enum. This became issue with 3.2 dev build.

```
# sort the table entries by a function metadata attribute
if column in self.METADATA_ATTRIBUTES:
    # FIXME:: Latest dev, PySide6.QtCore.Qt.SortOrder, enum 'SortOrder', enum is the problem
    sorted_functions = sorted(
                itervalues(self._visible_metadata),
                key=attrgetter(sort_field),
                reverse=sort_order
            )
  ```
  
  ```
  # sort the table entries by a function metadata attribute
if column in self.METADATA_ATTRIBUTES:
    # FIXME:: Latest dev, PySide6.QtCore.Qt.SortOrder, enum 'SortOrder', enum is the problem
    sorted_functions = sorted(
                itervalues(self._visible_metadata),
                key=attrgetter(sort_field),
                reverse=sort_order.value
            )
  ```

**Comment by AlxCzl** [comment]
Got the same issue, the fix is working on my PC.

**Comment by gaasedelen** [comment]
Thanks, fixed by 8f3c23bbb6f9bd825e70901688e0f98e7297b167
